### DISCUSSION TITLE
Cannot login to my account, with error message "Your account requires further verification. Please contact registration@vercel.com to complete your registration."

### DISCUSSION THREAD

**Discussion by hemvip** [vercel/community#7266] [Account Issue] [answered]
### Question

I got following error when login to vercel account. Please reply my email or comment.

![image](https://github.com/vercel/community/assets/171658074/58ef548a-1db8-410b-9a47-5606ab05a367)

My email account is hemvip@outlook.com

"Your account requires further verification. Please contact registration@vercel.com to complete your registration."

**Comment by swarnava** [comment, marked as answer]
Hi,

As the error message suggest, you need to contact registration@vercel.com so we can review your account.

**Reply by beemybold** [reply]
Hello there!
I'm facing the same challenges. Can you help me out please?

**Reply by dangvankhanhduy** [reply]
> Hello there! I'm facing the same challenges. Can you help me out please?
Tạo maill trước xong liên kết qua git lại là được.

[a comment by an automated account is not shown]
